lib/api/authn/force-user.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('../../debug').authentication
+
+/**
+ * Enforces the `--force-user` server flag, hardcoding a webid for all requests,
+ * for testing purposes.
+ */
+function initialize (app, argv) {
+  const forceUserId = argv.forceUser
+  app.use('/', (req, res, next) => {
+    debug(`Identified user (override): ${forceUserId}`)
+    req.session.userId = forceUserId
+    res.set('User', forceUserId)
```

Comment:
I don't think that's the case: the TLS handler is not attached in presence of `forceUser` (neither in my code nor the original): https://github.com/solid/node-solid-server/pull/531/commits/b13377fd68b6aed465eaeb9ec7dcda91a1287782#diff-237045f786d4918cf906b96a1c163eb9L203